src/swap/components/Swap.tsx

```diff
@@ -17,7 +17,7 @@ export function Swap({
     maxSlippage: DEFAULT_MAX_SLIPPAGE,
   },
   className,
-  experimental = { useAggregator: true },
+  experimental = { useAggregator: false },
```

Comment:
If we are switching, why keep all this extra code?

If we are confident in how it works, we should just remove all `experimental` code.

Reply:
Are we thinking to do that later? And later when?

Reply:
let's leave this option in for developers for now. will share the migration plan in Slack